tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html

```diff
@@ -441,10 +477,10 @@
                     if (draggedColPos > width) {
                       draggedColPos = width;
                     }
-                    cols.sort(function(a, b) {
+                    _this._cols.sort(function(a, b) {
```

Comment:
What happens if you sort the cols when you set `this_.cols` instead? This can create unwanted behavior elsewhere.